### DISCUSSION TITLE
Jupyter Notebook - AttributeError: module 'altair' has no attribute 'chart'

### DISCUSSION THREAD

**Discussion by tangs3352a** [vega/altair#3259] [Q&A] [answered]
Hi All,
Would someone be able to help me with Altair? I am using windows OS and Jupyter Notebook (open via cmd). 

When I try to use Altair to generate a graph I get the below error message.  
![image](https://github.com/altair-viz/altair/assets/143649826/4e298373-7ca0-47f7-8a2c-e6d23bf9956a)
![image](https://github.com/altair-viz/altair/assets/143649826/0171559a-818e-4ca5-97e1-7403f369b4eb)
![image](https://github.com/altair-viz/altair/assets/143649826/102ecf74-7c16-497c-b3a7-e84f905d0232)

**Comment by mattijn** [comment, marked as answer]
It's capital `C` -> `alt.Chart()`

**Reply by tangs3352a** [reply]
;____; THANK YOU!!!!!!!!!!!!
